.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description of change
+
+<!-- Briefly describe your changes here. If you are replacing an existing icon, provide a before and after screenshot -->
+
+### Link to demonstration
+
+<!-- Provide a link to a interactive map that uses the new icon with 11x11 and 15x15 variation. -->
+
+---
+
+Closes #<!--insert issue number here-->.
+
+### For author
+
+- [ ] Cite changes under the `HEAD` tag in [`CHANGELOG.md`](../CHANGELOG.md)
+- [ ] PR contains only one icon with 11x11 and 15x15 variation
+- [ ] Changes fall in accordance to the [Maki guidelines](https://labs.mapbox.com/maki-icons/guidelines/)
+  - [ ] Icon geometry is aligned to the pixel grid
+  - [ ] Icon stays within a 2px trim area
```

Comment:
Not sure where we landed on trim area. Is this still true?

Reply:
I think a _trim area_ is still true. for background fills we want padding space. Where we land on pixel dimension is still up for debate?

Reply:
True, I guess we can adjust as we experiment with dimensions. Thanks!